morph: deploy fails

**Describe the bug**
Morph doesn't work without ssh installed/present on the system.

**To Reproduce**
Steps to reproduce the behavior:
1. nix-shell --pure -p morph
2. morph build something

**Expected behavior**
Morph builds something

**Screenshots**

```
[nix-shell:/home/b1000101]$ morph build  
panic: Missing dependencies: nix, scp, ssh

goroutine 1 [running]:
main.init.0()
	/build/go/src/github.com/dbcdk/morph/morph.go:212 +0x104
```

**Additional context**

**Metadata**
NicksOhAss:~$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 4.19.97, NixOS, 19.09.1914.d14cea0dec2 (Loris)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3.2`
 - channels(root): `"nixos-19.09.1914.d14cea0dec2"`
 - channels(b1000101): `"unstable-20.03pre209574.2628f20267d"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`


Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute:
# a list of nixos modules affected by the problem
module:
```

--- a/pkgs/tools/package-management/morph/default.nix
+++ b/pkgs/tools/package-management/morph/default.nix
@@ -1,4 +1,4 @@
-{ buildGoPackage, fetchFromGitHub, go-bindata, lib }:
+{ buildGoPackage, fetchFromGitHub, go-bindata, openssh, makeWrapper, lib }:
 
 buildGoPackage rec {
   pname = "morph";
@@ -14,6 +14,7 @@ buildGoPackage rec {
   goPackagePath = "github.com/dbcdk/morph";
   goDeps = ./deps.nix;
 
+  nativeBuildInputs = [ makeWrapper ];
   buildInputs = [ go-bindata ];
 
   buildFlagsArray = ''
@@ -29,6 +30,7 @@ buildGoPackage rec {
   postInstall = ''
     mkdir -p $lib
     cp -v $src/data/*.nix $lib
+    wrapProgram $bin/bin/morph --prefix PATH : ${lib.makeBinPath [ openssh ]};
   '';
 
   outputs = [ "out" "bin" "lib" ];

